_posts/2018-08-13-Introducing....md

```diff
@@ -0,0 +1,96 @@
+
+#### “Tell me about your research at the Turing”
+
+Yini’s research interests at the Turing divides into three stems:
+
+1. Finding out how personality types relate to mental health differences: using questionnaires and scales to scope out people’s personalities or individual traits and then building a model to predict mental health issues when that person is under stress, compared to when they’re not.
+2. Finding out how neuroimaging can be used to predict mental health differences: investigating what are the biomarkers or the brain regions which suggest different mental states.
+3. Investigating the relationship between the personality, the brain, and mental health. Yini wants to find relationships between the personality and the brain, and how both of them could influence human behaviour and susceptibility to certain types of mental health issues. 
+
+Personality and the brain may be combined to influence behaviour. In research, Yini collects images of different people’s brains and data about their behaviour, and uses longitudinal data to examine these under different stress conditions to see how the brain and personality will influence mental health. Collecting data over multiple points in time is important, as it helps to discern causes from correlations. 
+
+#### “What’s your ideal outcome of your time at the Turing?”
+
+Yini is very interested in open science, which is new to her but which she thinks is a very good and clean way of doing research. She wants to study Kirstie’s way of doing research and learn from her. She also wants to improve her skills in coding and work with other people who can help her. She can learn from Kirstie, who is a very supportive supervisor, and she’s also happy to find peers and friends to learn new things with. For Yini, teamwork is very important, and at the Turing she can find good team members and support to help answer meaningful questions – which is her ideal outcome.
+
+#### “How are you finding the Turing?”
+
+“I think my major and interests involve very complex questions, based on a lot of different kinds of knowledge such as neuroimaging, computer programming, and biology, so I would like to communicate with people from different fields, and coming to the Turing I’ve met a lot of amazing people, so it’s very interesting.”
+
```

Comment:
Making friends with you is  a lucky thing in my life!!!

Reply:
Thanks Yini you too :D